content/realtime/channels/channel-parameters/overview.textile

```diff
@@ -0,0 +1,82 @@
+---
+title: Channel Parameters
+section: realtime
+index: 1
+jump_to:
+  Help with:
+    - Overview#overview
+    - Supported channel parameters#supported-parameters
+    - Using channel parameters#using-parameters
+    - Using channel parameters with non-Ably transports#using-parameters-with-other-transports
+    - Next steps#next-steps
+---
+
+h2(#overview). Overview
+
+Channel parameters allow a client to set the properties of a channel. A set of channel parameters is a set of key/value pairs, where both keys and values are strings; the keys correspond to specific features that Ably defines.
+
+The methods provided for specifying channel parameters are outlined below.
+
+h2(#supported-parameters). Supported channel parameters
+
+- rewind := Used to request that an attachment start from a given number of messages or point in time in the past. See "rewind":/realtime/channels/channel-parameters/rewind for more information.
+- delta := Used to request that data payloads should be sent as deltas to the previous payload. See "deltas":/realtime/channels/channel-parameters/deltas for more information.
+
+h2(#using-parameters). Using channel parameters
+
+You can specify channel parameters in the @ChannelOptions@ when obtaining a @Channel@. A collection of channel parameters is expressed as a map of string key/value pairs. The @ChannelOptions@ associated with a channel may also be updated by calling "setOptions":/realtime/channels/overview#modifying-options. The parameters associated with a channel take effect when the channel is first attached; if the parameters are subsequently modified via a call to @setOptions@, then that call triggers an attach operation that applies the updated parameters, if successful.
+
+h3. Example
+
+For example, to specify the @rewind@ channel param with the value @"1"@:
+
+```[javascript]
+  const realtime = new Ably.Realtime('{{API_KEY}}');
+  const channelOpts = {params: {rewind: '1'}}
+  const channel = realtime.channels.get('{{RANDOM_CHANNEL_NAME}}', channelOpts);
+```
+
+To modify the @rewind@ channel param with the value @"15s"@:
+
+```[javascript]
+  const realtime = new Ably.Realtime('{{API_KEY}}');
+  const channelOpts = {params: {rewind: '15s'}}
+  channel.setOptions(channelOpts, (err) => {
```

Comment:
I don't think `setOptions` is supported across the board.  We need to make that clear by SDK.

Reply:
@tomczoink another SDK specific ask.

Reply:
Think this may be more one for @QuintinWillison , not sure on usage for other languages unfortunately.

Reply:
Will resolve in a meeting.